frontend/src/container/MetricsApplication/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { TopOperationList } from './TopOperationsTable';
+
+export const getErrorCount = (list: TopOperationList): number =>
```

Comment:
I think function name should be `getErrorRate`

Reply:
good catch updated